Fails building:
```
In file included from ./core/object/method_bind.h:34,
                 from ./core/object/class_db.h:34,
                 from ./core/object/ref_counted.h:34,
                 from ./core/io/resource_uid.h:34,
                 from ./core/io/resource.h:34,
                 from ./core/input/input_event.h:35,
                 from ./core/input/shortcut.h:34,
                 from ./editor/editor_command_palette.h:34,
                 from editor/editor_command_palette.cpp:31:
./core/variant/binder_common.h: In instantiation of 'void call_with_variant_args_helper(T*, void (T::*)(P ...), const Variant**, Callable::CallError&, IndexSequence<Is ...>) [with T = EditorCommandPalette; P = {String&}; long unsigned int ...Is = {0}]':
./core/variant/binder_common.h:417:40:   required from 'void call_with_variant_args(T*, void (T::*)(P ...), const Variant**, int, Callable::CallError&) [with T = EditorCommandPalette; P = {String&}]'
./core/object/callable_method_pointer.h:104:25:   required from 'void CallableCustomMethodPointer<T, P>::call(const Variant**, int, Variant&, Callable::CallError&) const [with T = EditorCommandPalette; P = {String&}]'
./core/object/callable_method_pointer.h:100:15:   required from here
Error: ./core/variant/binder_common.h:303:25: error: cannot bind non-const lvalue reference of type 'String&' to an rvalue of type 'String'
  303 |  (p_instance->*p_method)(VariantCasterAndValidate<P>::cast(p_args, Is, r_error)...);
      |  ~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/editor/editor_command_palette.cpp
+++ b/editor/editor_command_palette.cpp
@@ -34,6 +34,7 @@
 #include "editor/editor_scale.h"
 #include "editor/editor_settings.h"
 #include "editor/editor_string_names.h"
+#include "editor/gui/editor_toaster.h"
 #include "scene/gui/control.h"
 #include "scene/gui/tree.h"
 
@@ -184,10 +185,10 @@ void EditorCommandPalette::_sbox_input(const Ref<InputEvent> &p_ie) {
 
 void EditorCommandPalette::_confirmed() {
 	TreeItem *selected_option = search_options->get_selected();
-	String command_key = selected_option != nullptr ? selected_option->get_metadata(0) : "";
+	const String command_key = selected_option != nullptr ? selected_option->get_metadata(0) : "";
 	if (!command_key.is_empty()) {
 		hide();
-		execute_command(command_key);
+		callable_mp(this, &EditorCommandPalette::execute_command).call_deferred(command_key);
 	}
 }
 
@@ -248,11 +249,19 @@ void EditorCommandPalette::_add_command(String p_command_name, String p_key_name
 	commands[p_key_name] = command;
 }
 
-void EditorCommandPalette::execute_command(String &p_command_key) {
+void EditorCommandPalette::execute_command(const String &p_command_key) {
 	ERR_FAIL_COND_MSG(!commands.has(p_command_key), p_command_key + " not found.");
 	commands[p_command_key].last_used = OS::get_singleton()->get_unix_time();
-	commands[p_command_key].callable.call_deferred();
 	_save_history();
+
+	Variant ret;
+	Callable::CallError ce;
+	const Callable &callable = commands[p_command_key].callable;
+	callable.callp(nullptr, 0, ret, ce);
+
+	if (ce.error != Callable::CallError::CALL_OK) {
+		EditorToaster::get_singleton()->popup_str(vformat(TTR("Failed to execute command \"%s\":\n%s."), p_command_key, Variant::get_callable_error_text(callable, nullptr, 0, ce)), EditorToaster::SEVERITY_ERROR);
+	}
 }
 
 void EditorCommandPalette::register_shortcuts_as_command() {

--- a/editor/editor_command_palette.h
+++ b/editor/editor_command_palette.h
@@ -97,7 +97,7 @@ public:
 	void open_popup();
 	void get_actions_list(List<String> *p_list) const;
 	void add_command(String p_command_name, String p_key_name, Callable p_action, Vector<Variant> arguments, String p_shortcut_text = "None");
-	void execute_command(String &p_command_name);
+	void execute_command(const String &p_command_name);
 	void register_shortcuts_as_command();
 	Ref<Shortcut> add_shortcut_command(const String &p_command, const String &p_key, Ref<Shortcut> p_shortcut);
 	void remove_command(String p_key_name);

